07-workbase/01-preferences.md

```diff
@@ -5,132 +5,68 @@ permalink: /docs/workbase/connection
-[footer:start]Click on the gear icon to open the preferences panel.[footer:end]
-[slide:end]
-
-[slide:start]
-[header:start][header:end]
-[body:start]![Launch](/images/workbase/preferences_b.png)[body:end]
-[slide:end]
-
-[slide:start]
-[header:start][header:end]
-[body:start]![Launch](/images/workbase/preferences_keyspaces_a.png)[body:end]
-[footer:start]The bottom part of the preferences panel allows us to view, create and delete keyspaces.[footer:end]
-[slide:end]
-
-[slide:start]
-[header:start][header:end]
-[body:start]![Launch](/images/workbase/preferences_keyspaces_b.png)[body:end]
-[footer:start]To create a new keyspace, we need to enter a name and click on **Create New Keyspace**.[footer:end]
-[slide:end]
-
-[slide:start]
-[header:start][header:end]
-[body:start]![Launch](/images/workbase/preferences_keyspaces_c.png)[body:end]
-[footer:start]This creates the new keyspace and displays it in the list.[footer:end]
-[slide:end]
-
-[slide:start]
-[header:start][header:end]
-[body:start]![Launch](/images/workbase/preferences_keyspaces_d.png)[body:end]
-[footer:start]To delete a keyspace, we need to click on the _trash_ icon next to it.[footer:end]
-[slide:end]
-
-[slide:start]
-[header:start][header:end]
-[body:start]![Launch](/images/workbase/preferences_keyspaces_a.png)[body:end]
-[footer:start][footer:end]
+[body:start]![Select a Keyspace](/docs/images/workbase/1.1.1/keyspaces_1.png)[body:end]
+[footer:start]To select a keyspace for Workbase to connect to, we need to click on the keyspace button.[footer:end]
 [slide:end]
 
-</div> -->
-
-## Connect to Grakn
-Workbase connects to a running [Grakn Server](/docs/running-grakn/install-and-run#start-the-grakn-server) and interacts with [keyspaces](/docs/management/keyspace). In this short section, we learn how to configure this connection and select a keyspace to interact with.
-
-### Specify the host and port
-<div class="slideshow">
-
 [slide:start]
-[body:start]![Specify the host and port](/docs/images/workbase/main_settings.png)[body:end]
-[footer:start]Right after opening the Workbase, click on the _gear_ icon in the right sidebar.[footer:end]
+[body:start]![Select a Keyspace](/docs/images/workbase/1.1.1/keyspaces_2.png)[body:end]
+[footer:start]This shows us the list of all keyspaces running on the Grakn Server. We then select the keyspace of interest.[footer:end]
 [slide:end]
 
 [slide:start]
-[header:start][header:end]
-[body:start]![Specify the host and port](/docs/images/workbase/settings_connection.png)[body:end]
-[footer:start]Under _Connection Settings_ , alter the host and port as per your [custom configuration of Grakn Server](/docs/running-grakn/configuration##host-and-port).[footer:end]
+[body:start]![Select a Keyspace](/docs/images/workbase/1.1.1/keyspaces_3.png)[body:end]
+[footer:start]Keyspace is now connected to the selected keyspace, in our case that is named `social_network`.[footer:end]
 [slide:end]
 
-</div>
-
-### Select a Keyspace
```

Comment:
**Select a Keyspace** and **Manage Keyspaces** are two separate use cases and so should be documented that way. my line of reasoning is, that selecting a keyspace isn't a part of managing keyspaces. it is an action that's required to start interacting with that keyspace.